orchestrator/src/main/kotlin/Orchestrator.kt

```diff
@@ -340,9 +332,22 @@ class Orchestrator(
 
         val ortRun = getCurrentOrtRun(ortRunId)
         val scheduleContext = createWorkerSchedulerContext(ortRun, header, workerJobs = jobs)
-        val schedules = WorkerScheduleInfo.entries.mapNotNull { it.createAndScheduleJobIfPossible(scheduleContext) }
 
-        return scheduleContext to schedules
+        return fetchNextJobs(scheduleContext)
```

Comment:
how about `scheduleContext to fetchNextJobs(scheduleContext)` ?

Then you would not have to return a `pair` from the function.

Reply:
This makes a construct easier that is used in the next commit.